ckcp/docker/entry.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+rm -rf .kcp/
+./kcp/bin/kcp start \
+  --push-mode=true \
+  --pull-mode=false \
+  --install-cluster-controller \
+  --install-workspace-controller \
+  --auto-publish-apis \
+  --resources-to-sync="deployments.apps,pods,services" >> kcp.log 2>&1
```

Comment:
You can remove the ` >> kcp.log 2>&1` part. Logs will go to the k8s logs destination.

You will be able to see logs with kubectl logs.

Reply:
Done.